components/Frame/Footer.js

```diff
@@ -152,7 +152,7 @@ class Footer extends Component {
               <a
                 href='https://www.google.ch/maps/place/Sihlhallenstrasse+1,+8004+Zürich'
                 target='_blank'
-                rel='noopener'
+                rel='noopener noreferrer'
```

Comment:
we already set the meta tag in `pages/_document.js`:
```
<meta name='referrer' content='no-referrer' />
```
And the `no-referrer` header via `herlemt` in `server/index.js`. If the linter wants this: disable the rule.